sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/implementation/models/ChatCompletionsOptions.java

```diff
@@ -0,0 +1,534 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+package com.azure.ai.openai.implementation.models;
```

Comment:
This class should be in `com.azure.ai.openai.models`, it is a public accessible class.

Reply:
We still had the handwritten one in models.

This one in implementation because of the Union on function.
https://github.com/Azure/azure-sdk-for-java/blob/a0f3e852e5b5019a0a51ad9d9a6a8c0d8993c89a/sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/implementation/models/ChatCompletionsOptions.java#L40-L48

We hide it in the public class
https://github.com/Azure/azure-sdk-for-java/blob/a0f3e852e5b5019a0a51ad9d9a6a8c0d8993c89a/sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/ChatCompletionsOptions.java#L455-L458